src/sentry/api/endpoints/project_codeowners.py

```diff
@@ -154,7 +154,7 @@ def get(self, request: Request, project: Project) -> Response:
         expand = request.GET.getlist("expand", [])
         expand.append("errors")
 
-        codeowners = list(ProjectCodeOwners.objects.filter(project=project))
+        codeowners = list(ProjectCodeOwners.objects.filter(project=project).order_by("-date_added"))
```

Comment:
I don't think we need to solve it yet, but there might be a perf issue here of `date_added` is not indexed.